roles/kubernetes-apps/ansible/templates/dashboard.yml.j2

```diff
@@ -127,18 +112,10 @@ spec:
       labels:
         k8s-app: kubernetes-dashboard
     spec:
-{% if not dashboard_use_custom_certs %}
```

Comment:
We still need this to allow cert overrides.

Reply:
ok